.github/workflows/conference.yml

```diff
@@ -53,3 +53,41 @@ jobs:
           labels: |
             report
             automated pr
+  add_comment:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ env.GITHUB_TOKEN }}
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Install dependencies
+        run: pip install PyGithub PyYAML pre-commit
+
+      - name: Run script
+        env:
+          GITHUB_TOKEN: ${{ env.GITHUB_TOKEN }}
+        run: |
+          python _conferences
+          pre-commit run --all-files --show-diff-on-failure
+
+      - name: Add comment to the issue
+        uses: actions/github-script@v6
+        if: ${{ github.event.issue.user.type == 'User' && startsWith(github.event.issue.title, '[CONFERENCE]') && contains(github.event.issue.labels.*.name, 'conference') }} #adds comment only if the issue is created by a user
```

Comment:
I think you can remove the `{{ github.event.issue.user.type == 'User'`

and instead use the "event.action" to be "issue opened" (Not sure what the actual phrase is.

We don't want it to run that las step on edits and closing, when the issue if created.

Reply:
Noted will work on it